src/screens/Exchange/Exchange.js

```diff
@@ -730,6 +732,11 @@ class ExchangeScreen extends React.Component<Props, State> {
                 </ButtonLabel>
               </CardButton>
               }
+              {!!isFiat && !isAllowed &&
```

Comment:
I think this `CardButton` should be disabled, as well other button with amount should be also applied with logic to make it hidden.